examples/kafka/dev-values-kafka-proxy.yaml

```diff
@@ -161,7 +161,7 @@ proxy:
 # The Proxy uses the same TLS certificate configured for the Pulsar protocol
 #    PULSAR_PREFIX_kafkaAdvertisedListeners: "SASL_PLAINTEXT://pulsar-proxy:9092,SASL_SSL://pulsar-proxy:9093"
 # if you use port forwarding to localhost then you have to set 'localhost'    
-    PULSAR_PREFIX_kafkaAdvertisedListeners: "SASL_PLAINTEXT://localhost:9092,SASL_SSL://localhost:9093"
+    PULSAR_PREFIX_kafkaAdvertisedListeners: "SASL_PLAINTEXT://pulsar-proxy:9092,SASL_SSL://pulsar-proxy:9093"
```

Comment:
please do not change this example
this is supposed to work with port-forwarding

actually we could add two listeners, one that advertises itself on "localhost" and one on "pulsar-proxy", on 2 different ports, this way it will be accessible both via port-forwarding and from the k8s cluster
but I would keep this change as a separate PR